index.html

```diff
@@ -3587,13 +3587,18 @@ <h5><code>subscribeevent</code></h5>
 	  <span class="rfc2119-assertion" 
             id="http-sse-profile-protocol-binding-subscribeevent-6d">
             The <code>id</code> field SHOULD be set to a unique identifier for 
```

Comment:
Is it permitted to have two events with the same id and different notification content?

Reply:
No, that's what "unique identifier" means :)

Event affordances emit multiple events with the same event name but with a unique identifier and potentially different notification content in each.